src/Uno.UWP/ApplicationModel/Package.cs

```diff
@@ -23,8 +25,21 @@ internal Package()
 		[Uno.NotImplemented]
 		public IReadOnlyList<Package> Dependencies => new List<Package>();
 
-		public StorageFolder InstalledLocation
-			=> new StorageFolder(GetInstalledLocation());
+		/// <summary>
+		/// Gets the current package's location in the original install folder for the current package.
+		/// </summary>
+#if __WASM__
+		[Uno.NotImplemented("__WASM__")]
+#endif
```

Comment:
Why doesn't this implementation work for Wasm?

Reply:
The `InstalledPath` is useful mainly to build up path to assets within the app package, which in this case is not really possible, as you would need an URL instead of a filesystem path. Hence, I mark it as not implemented as it does not really fulfill the intended purpose.